plotting.py

```diff
@@ -114,6 +116,70 @@ def plot_inputs(data, particles_dict):
 			particles_dict
 			)
 
+def plot_NN(yhat, data):

```

Comment:
this function doesn't actually plot the neural network (one may think it produces a graph of its structure...). Maybe find a more descriptive name for what this function actually does.